spec/models/manageiq/providers/amazon/cloud_manager/refresher_spec.rb

```diff
@@ -7,6 +7,10 @@
     @ems.update_authentication(:default => {:userid => "0123456789ABCDEFGHIJ", :password => "ABCDEFGHIJKLMNO1234567890abcdefghijklmno"})
   end
 
+  it ".ems_type" do
+    expect(described_class.ems_type).to eq(:ec2)
+  end
```

Comment:
We can let this in for now, but this seems expensive because the before block will run for basically no reason.  @blomquisg We should clean this up in a separate PR by perhaps putting full refresh specs in their own files.